js/client-battle-tooltips.js

```diff
@@ -879,7 +880,7 @@ var BattleTooltips = (function () {
 		}
 		// Other abilities that change the move type.
 		if ('sound' in move.flags && ability === 'Liquid Voice') moveType = 'Water';
-		if (moveType === 'Normal' && move.category && move.category !== 'Status' && !(move.id in {'naturalgift': 1, 'struggle': 1})) {
+		if (moveType === 'Normal' && move.category && move.category !== 'Status' && !(move.id in {judgment:1, multiattack:1, naturalgift:1, revelationdance:1, technoblast:1, weatherball:1})) {
```

Comment:
Struggle still needs to be in this list and the one below. The server doesn't need it because Struggle is ??? type by the time it gets to that point, so isn't affected anyway.

Reply:
Whoops, didn't realize struggle got removed there. Also, are judgment, etc. still affected by -ate in gen 6? Cuz if they are I need to change this

Reply:
Nah, same behaviour.